spec/controllers/idv/questions_controller_spec.rb

```diff
@@ -0,0 +1,43 @@
+require 'rails_helper'
+
+describe Idv::QuestionsController do
+  render_views
+
+  let(:user) { create(:user, :signed_up, email: 'old_email@example.com') }
+  let(:applicant) { Proofer::Applicant.new first_name: 'Some', last_name: 'One' }
+  let(:agent) { Proofer::Agent.new vendor: :mock }
+  let(:resolution) { agent.start applicant }
+
+  context 'user has started proofing session' do
+    it 'requires user is signed in' do
```

Comment:
Instead of checking this for every action we have been using the check for the before_filter in the controller.  (grep for  have_filters in the specs tree)